### PR TITLE
Changed ee4 instances to ee

### PR DESCRIPTION
**Pull request by kirtangajjar**
Some instances have been intentionally left out.

1. `ee4-config` folder - It'll be removed in https://github.com/EasyEngine/easyengine/pull/1040
2. `php/class-ee-docker.php` - It'll be refactored in https://github.com/EasyEngine/easyengine/pull/1041

### CODE REVIEW

File: php/EE/Runner.php [hunk 1]
```diff
@@ -555,7 +556,7 @@ private function ensure_present_in_config( $var, $default) {
 		if ( empty($this->config[$var]) ) {
 			$this->config[$var] =  $default ;
 
-			$config_file_path = getenv('EE_CONFIG_PATH') ? getenv('EE_CONFIG_PATH') : Utils\get_home_dir() . '/.ee4/config.yml';
+			$config_file_path = getenv('EE_CONFIG_PATH') ? getenv('EE_CONFIG_PATH') : EE_CONFIG_PATH . '/config.yml';
```
Comment (mrrobot47) [inline php/EE/Runner.php]: Using wrong constant. Use `EE_CONF_ROOT`

File: php/EE/Runner.php [hunk 2]
```diff
@@ -601,7 +602,7 @@ private function run_alias_group( $aliases ) {
 		if ( getenv( 'EE_CONFIG_PATH' ) ) {
 			$config_path = getenv( 'EE_CONFIG_PATH' );
 		} else {
-			$config_path = Utils\get_home_dir() . '/.ee4/config.yml';
+			$config_path = EE_CONFIG_PATH . '/config.yml';
```
Comment (mrrobot47) [inline php/EE/Runner.php]: Using wrong constant. Use `EE_CONF_ROOT`

File: php/EE/Runner.php [hunk 3]
```diff
@@ -629,10 +630,10 @@ private function set_alias( $alias ) {
 
 	public function start() {
 
-		$this->ensure_present_in_config( 'sites_path', Utils\get_home_dir(). '/ee4-sites' );
-		$this->ensure_present_in_config( 'db_path', Utils\get_home_dir(). '/.ee4/ee4.db' );
+		$this->ensure_present_in_config( 'sites_path', Utils\get_home_dir(). '/ee-sites' );
+		$this->ensure_present_in_config( 'db_path', EE_CONFIG_PATH. '/ee.db' );
```
Comment (mrrobot47) [inline php/EE/Runner.php]: Using wrong constant. Use `EE_CONF_ROOT`

File: php/class-ee.php [hunk 4]
```diff
@@ -843,7 +843,7 @@ public static function launch_self( $command, $args = array(), $assoc_args = arr
 		if ( getenv( 'EE_CONFIG_PATH' ) ) {
 			$config_path = getenv( 'EE_CONFIG_PATH' );
 		} else {
-			$config_path = Utils\get_home_dir() . '/.ee4/config.yml';
+			$config_path = EE_CONFIG_PATH . '/config.yml';
```
Comment (mrrobot47) [inline php/class-ee.php]: Using wrong constant. Use `EE_CONF_ROOT`

File: php/commands/src/CLI_Command.php [hunk 5]
```diff
@@ -187,7 +187,7 @@ public function info( $_, $assoc_args ) {
 	 */
 	public function update( $_, $assoc_args ) {
 
-		$config_file_path = getenv( 'EE_CONFIG_PATH' ) ? getenv( 'EE_CONFIG_PATH' ) : Utils\get_home_dir() . '/.ee4/config.yml';
+		$config_file_path = getenv( 'EE_CONFIG_PATH' ) ? getenv( 'EE_CONFIG_PATH' ) : EE_CONFIG_PATH . '/config.yml';
```
Comment (mrrobot47) [inline php/commands/src/CLI_Command.php]: Using wrong constant. Use `EE_CONF_ROOT`

File: php/EE/Runner.php [hunk 6]
```diff
@@ -119,7 +119,8 @@ public function get_global_config_path() {
 			$config_path = getenv( 'EE_CONFIG_PATH' );
 			$this->_global_config_path_debug = 'Using global config from EE_CONFIG_PATH env var: ' . $config_path;
 		} else {
-			$config_path = Utils\get_home_dir() . '/.ee4/config.yml';
+			// $config_path = Utils\get_home_dir() . '/.ee/config.yml';
+			$config_path = EE_CONFIG_PATH . '/config.yml';
```
Comment (mrrobot47) [inline php/EE/Runner.php]: Using wrong constant. Use `EE_CONF_ROOT`.

File: php/EE/Runner.php [hunk 7]
```diff
@@ -119,7 +119,8 @@ public function get_global_config_path() {
 			$config_path = getenv( 'EE_CONFIG_PATH' );
 			$this->_global_config_path_debug = 'Using global config from EE_CONFIG_PATH env var: ' . $config_path;
 		} else {
-			$config_path = Utils\get_home_dir() . '/.ee4/config.yml';
+			// $config_path = Utils\get_home_dir() . '/.ee/config.yml';
```
Comment (mrrobot47) [inline php/EE/Runner.php]: Remove unnecessary comments.

**Comment by kirtangajjar** [comment]
@mrrobot47 Done.

File: composer.json [hunk 8]
```diff
@@ -77,5 +77,11 @@
             "split-target-prefix-true": "autoload_commands",
             "split-target-prefix-false": "autoload_framework"
         }
+    },
+    "repositories": {
```
Comment (mrrobot47) [inline composer.json]: Only add and commit the files you are working on. Strictly avoid using `git add --all` or `git add .`

Remove this file from the PR.

File: composer.lock [hunk 9]
```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#composer-lock-the-lock-file",
         "This file is @generated automatically"
     ],
-    "content-hash": "80bc4a48c5afd7e3306ace1fb21e17e3",
+    "content-hash": "dcc98937f8f442d4cdaae1c164b7eed8",
```
Comment (mrrobot47) [inline composer.lock]: Remove your `composer.lock` from PR.

File: php/EE/Runner.php [hunk 10]
```diff
@@ -173,7 +173,7 @@ public function get_packages_dir_path() {
 		if ( getenv( 'EE_PACKAGES_DIR' ) ) {
 			$packages_dir = Utils\trailingslashit( getenv( 'EE_PACKAGES_DIR' ) );
 		} else {
-			$packages_dir = Utils\get_home_dir() . '/.ee4/packages/';
+			$packages_dir = Utils\get_home_dir() . '/.ee/packages/';
```
Comment (mrrobot47) [inline php/EE/Runner.php]: Shouldn't the packages also go in the common global easyengine config directory?

Comment (rahul286) [reply php/EE/Runner.php]: home dir packages folder will only make sense if we want to support per user package.

but default should be `/opt/easyengine/packages/`

File: php/EE/Runner.php [hunk 11]
```diff
@@ -47,19 +47,24 @@ public function __get( $key ) {
...
+	 * Function to check and create the root directory for ee.
 	 */
-	private function init_ee4() {
+	private function init_ee() {
 
 		if ( ! is_dir( EE_CONF_ROOT ) ) {
-			mkdir( EE_CONF_ROOT );
+			$user = getenv('USER');
+			shell_exec('sudo mkdir -p ' . EE_CONF_ROOT);
+			shell_exec("sudo chown -R $user: " . EE_CONF_ROOT);
```
Comment (mbtamuli) [inline php/EE/Runner.php]: Please remove this `chown` as we'll be tackling multi user usage of EasyEngine in later release.

File: php/EE/Runner.php [hunk 12]
```diff
@@ -480,14 +485,23 @@ public function init_logger() {
 
 		EE::set_logger( $logger );
 
+		$this->init_ee();
+
+		if ( !empty( $this->arguments[0] ) && 'cli' === $this->arguments[0] && ! empty( $this->arguments[1] ) && 'info' === $this->arguments[1] && ! $this->config['ssh'] ) {
+			$file_logging_path = '/dev/null';
+		}
+		else {
+			$file_logging_path = EE_CONF_ROOT . '/ee4.log';
```
Comment (mbtamuli) [inline php/EE/Runner.php]: Missed a line of ee4.

**Pull request merged**